products/BellHybrid/services/appmgr/ApplicationManager.cpp

```diff
@@ -65,5 +66,6 @@ namespace app::manager
         connect(typeid(AlarmDeactivated), convertibleToActionHandler);
         connect(typeid(BatteryShutdown), convertibleToActionHandler);
         connect(typeid(BedtimeNotification), convertibleToActionHandler);
+        connect(typeid(ChangeHomescreenLayoutMessage), convertibleToActionHandler);
```

Comment:
Why it's there? I cannot see the usage.

Reply:
This commit contains only "backend" part of homescreen changes. Setting new homescreen ("frontent" part) will is implemented in separate task BH-1314